integration/headers-test.ts

```diff
@@ -12,193 +13,196 @@ test.describe("headers export", () => {
   let appFixture: Fixture;
 
   test.beforeAll(async () => {
```

Comment:
Ignore changes in here, I added ServerMode.Test to suppress some thrown response logging, and the rest is whitespace changes for the updated indentation